testdata/go/v3/monitoring/memcached-operator/Makefile

```diff
@@ -256,3 +256,14 @@ catalog-build: opm ## Build a catalog image.
 .PHONY: catalog-push
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
+
+##@ Generate the metrics documentation
+.PHONY: generate-metricsdocs
+generate-metricsdocs: build-metricsdocs
+	mkdir -p $(shell pwd)/docs/monitoring
+	_out/metricsdocs > docs/monitoring/metrics.md
+
+##@ Build the metrics documentation
+.PHONY: build-metricsdocs
+build-metricsdocs:
+	go build -ldflags="${LDFLAGS}" -o _out/metricsdocs ./monitoring/metricsdocs
```

Comment:
edit the generator to have an empty line at the end of file here

Reply:
Done dor both Makefile files and in the generator.